lib/node_modules/@stdlib/symbol/match/REDEME.md

```diff
@@ -0,0 +1,143 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# MatchSymbol
+
+> Match [symbol][mdn-symbol] which is used to determine if a constructor object recognizes an object as its instance.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var MatchSymbol = require( '@stdlib/symbol/match' );
+```
+
+#### MatchSymbol
+
+Match [`symbol`][mdn-symbol] which is used to determine if a constructor object recognizes an object as its instance.
+
+```javascript
+var s = typeof MatchSymbol;
+// e.g., returns 'symbol'
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   The [symbol][mdn-symbol] is only supported in environments which support [symbols][mdn-symbol]. In non-supporting environments, the value is `null`.
+
+-   The `instanceof` operator uses the following algorithm to determine the return value of `object instanceof constructor`:
+
+    -   If `constructor` has a `[MatchSymbol]()` method, the `instanceof` operator calls the method with `object` as the first argument and returns the result (coerced to a boolean). If `constructor` is not an object or if `constructor[MatchSymbol]` is neither `null`, `undefined`, nor a function, the `instanceof` operator raises an exception.
+    -   Otherwise, if `constructor` does not have a `[MatchSymbol]()` method (i.e., `constructor[MatchSymbol]` is `null` or `undefined`), the `instanceof` operator determines the result using the same algorithm as `Function.prototype[MatchSymbol]()`. If `constructor` is not a function, the `instanceof` operator raises an exception.
```

Comment:
This is all incorrect. You need to update to reflect the actual usage of the match symbol.